src/components/internals/calendar-header/calendar-header.mod.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  display: flex;
+  padding: 10px 2% 6px;
+  margin-bottom: 4px;
+  justify-content: space-between;
+  border-bottom: 1px solid #eaeaea;
+  align-items: center;
+}
+
+.month {
+  display: flex;
+}
+
+.month > * {
+  margin: 0 4px 0 0;
+}
+
+.year {
+  display: flex;
+}
+
+.year > * {
+  margin: 0 0 0 4px;
```

Comment:
I am such a party pooper! Why not 6px?

Reply:
Both 4px and 6px are part of our design system. Why would you prefer 6px?